execution/executor/src/db_bootstrapper.rs

```diff
@@ -54,6 +58,14 @@ pub fn bootstrap_db<V: VMExecutor>(
         .validators()
         .as_ref()
         .ok_or_else(|| format_err!("Genesis transaction must emit a validator set."))?;
+    let timestamp_usecs = if new_genesis_version == 0 {
+        // TODO(aldenhu): fix existing tests before using real timestamp
+        GENESIS_TIMESTAMP_USECS
```

Comment:
What's the problem here?

Reply:
tests are using `LedgerInfo::genesis()` which is without the non-zero timestamp.